concepts/observability-strategy/configurable-logging/01-logging-groundwork.md

```diff
@@ -4,9 +4,7 @@
 
 ![a](./assets/logging-current.drawio.svg)
 
-In the current setup, all three observability aspects (log, trace, metric) provide a preconfigured backend with visualisations. However, they don't provide a neutral and unified way to integrate backends outside of the cluster.
-
-The tracing stack does not support centrally pushing trace data to the outside.  Logging can be configured much more flexibly and neutrally. However, users must apply the configuration during installation; otherwise it's lost at the next Kyma upgrade process. Furthermore, it is hard to mix and match different integrations, because you must deal with one centralized configuration (the Fluent Bit config).
+In the current setup, the logging component provides feature-rich e2e logging solution with Loki as backend and Grafana as log Browser. The used log collector (FluentBit) can be configured flexibly and neutrally. However, users must apply the configuration during installation; otherwise it's lost at the next Kyma upgrade process. Furthermore, it is hard to mix and match different integrations, as you must deal with one centralized configuration (the Fluent Bit config) at deploy time. Furthermore, Loki is provided in a non-scalabale setup and cannot configured at runtime.
```

Comment:
```suggestion
In the current setup, the logging component provides a feature-rich end-to-end logging solution with Loki as backend and Grafana as log browser. Users can configure Fluent Bit, the log collector, flexibly and neutrally - but only during installation; otherwise it's lost at the next Kyma upgrade. Mixing and matching different integrations is hard, because you must deal with one centralized configuration (the Fluent Bit config) at deployment.
Furthermore, Kyma provides Loki in a non-scalabale setup, which users cannot configured at runtime.
```